go.mod

```diff
@@ -3,7 +3,7 @@ module github.com/terraform-providers/terraform-provider-aws
 go 1.14
 
 require (
-	github.com/aws/aws-sdk-go v1.34.4
+	github.com/aws/aws-sdk-go v1.36.0
```

Comment:
It looks like the go.mod and go.sum changes aren't necessary for this change, so they should be removed from the PR.

Reply:
Done

Reply:
Latest version of AWS SDK is needed for DPDTimeoutAction, StartupAction, LocalIpv6NetworkCidr, RemoteIpv6NetworkCidr, LocalIpv4NetworkCidr, RemoteIpv4NetworkCidr support.